composer.json

```
@@ -18,7 +18,7 @@
 		"pd/coding-standard": "1.17.*",
 		"nette/tester": "2.3.*",
 		"nette/bootstrap": "3.0.*",
-		"phpstan/phpstan-strict-rules": "0.12.*",
+		"phpstan/phpstan-strict-rules": "0.12.9",
```

```suggestion
		"phpstan/phpstan-strict-rules": "0.12.80",
```
Není to typo taková nízká verze?

---

je to verze balíku phpstan-strict-rules, ale přidal jsem explicitně i phpstan samotný